doc/user_guide/user-guide.md

```diff
@@ -0,0 +1,56 @@
+# Exasol REST API
+
+## Installation
+
+### Database Side Preparation
+
+* Create a user in the Exasol database for the api service. This user will be used for all connections. Grant **
+  minimum** privileges to the user. We strongly recommend to only grant SELECT on required schemas/tables. Please keep
+  in mind that all API users uses the same service account.
+
+```sql
+CREATE USER api_service_account IDENTIFIED BY secret_password;
```

Comment:
This is how we discussed it with Sebastian. Service account definition also fits into  the idea of this "user" perfectly:
"Service accounts are a special type of non-human privileged account used to execute applications and run automated services"

Reply:
In the quote: 
> Service accounts are a special type [...] account

So it only makes sense if there is already the concept of an account. So maybe api_service_user since in Exasol accounts are called user?

Reply:
I'll try improving the description, but I think it's better to make it clear that the user we create is a service account for the REST API application. A service account is a common term, it's better to say it explicitly that avoiding this term and describing that our user is not a real user but a user for a service

Reply:
"Service account" is also in our requirements and design documents